telemetry/prometheus/metrics.go

```diff
@@ -250,3 +253,13 @@ func forceValidName(name string) string {
 
 	return string(runes)
 }
+
+func setDefaultCfg(cfg *Config) {
+	// Set default values if not provided
```

Comment:
nit: move this comment to above the function signature.

Reply:
Moved. Good catch!